common/lib/capa/capa/templates/chemicalequationinput.html

```diff
@@ -10,8 +10,8 @@
    />
 
       <p class="status" aria-describedby="input_${id}">
-        ${value|h} -
-        ${status.display_name}
+        ${value|h}
```

Comment:
This is a bit funky with value being repeated again, but given that it was already this way and the problem type is unsupported, fine leaving it as-is.

http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/chemical_equation.html